### ISSUE TITLE
FCMEvent.Notification error in timeout...

### ISSUE THREAD

**Issue by Doskulov** [evollu/react-native-fcm#752]
I cloned this repo, and runned it to android emulator. I really dont know how to explain this, but i will try. So firstly it works clearly, every notification come. But after 2-3 minutes, any notifications dont work, except local. But after also 2-5 minutes, all notifications come 
![image](https://user-images.githubusercontent.com/15268684/35192766-4a17c34e-fec2-11e7-8b2a-ee64acfccc2d.png). So how can i solve this problem??

**Comment by evollu** [comment]
anything to do with google's notification design? if you send too often you get throttled?
There is no code here to delay the notification

**Comment by Doskulov** [comment]
@evollu no, there are problem with handling, i guess. Maybe this is the problem of firebase, or emulator or whatever. There is no problem with design, i just want to get any notification, which comes to my App. So i'm sending [https://fcm.googleapis.com/fcm/send](url), sometimes i'm getting, sometimes not. this is fully your code>..

**Comment by evollu** [comment]
try reproduce with Firebase's native example code. if you can reproduce it you should send a support ticket to them.
